library/std/src/sys_common/net.rs

```diff
@@ -328,6 +328,16 @@ impl TcpStream {
         self.inner.nodelay()
     }
 
+    #[cfg(target_os = "linux")]
+    pub fn set_quickack(&self, quickack: bool) -> io::Result<()> {
+        self.inner.set_quickack(quickack)
+    }
+
+    #[cfg(target_os = "linux")]
+    pub fn quickack(&self) -> io::Result<bool> {
+        self.inner.quickack()
+    }
+
```

Comment:
The `sys_common` module isn't supposed to contain code behind platform-specifc `cfg`s, instead the `linux::net` module should do enough unwrapping steps to get to the `os::unix` types directly.

Reply:
I added a to_inner method to expose the Socket